cohere/client.py

```diff
@@ -144,6 +144,33 @@ def choose_best(self, model: str, query: str, options: List[str], mode:  str = '
         response = self.__request(json_body, cohere.CHOOSE_BEST_URL, model)
         return BestChoices(response['scores'], response['tokens'], response['token_log_likelihoods'], mode)
 
+    def classify(
+        self,
+        model: str,
+        inputs: List[str],
+        examples: List[ClassifyExample] = List[None],
```

Comment:
just to understand this correctly? examples is an optional arg for the user right?

Reply:
hm actually rn it's not, will get rid of the default arg